docs/tutorial/file.rst

```diff
@@ -104,6 +104,11 @@ The channels have a few convenient methods, like `.plot()` which make it easy to
     f1x_timestamps = file.force1x.timestamps
     plt.plot(f1x_timestamps, f1x_data)
 
+The time values in seconds can also be accessed directly::
+
+    f1x_time = file.force1x.time
+    plt.plot(f1x_time, f1x_data)
```

Comment:
It might be also nice to quickly mention here that `.timestamps` returns absolute values in ns.